cmd/httpclient/main.go

```diff
@@ -62,35 +60,41 @@ func mainfunc() (err error) {
 	client := httpx.NewClient(handlers.StdoutHandler)
 	if *common.FlagHelp {
 		flag.CommandLine.SetOutput(os.Stdout)
-		fmt.Printf("Usage: httpclient [flags]\n")
+		fmt.Printf("Usage: httpclient [flags] <url>\n")
 		flag.PrintDefaults()
 		fmt.Printf("\nExamples:\n")
-		fmt.Printf("%s\n", "  ./httpclient -dns-transport system ...")
-		fmt.Printf("%s\n", "  ./httpclient -dns-transport godns ...")
-		fmt.Printf("%s\n", "  ./httpclient -dns-transport doh ...")
-		fmt.Printf("%s\n", "  ./httpclient -dns-transport dot ...")
-		fmt.Printf("%s\n", "  ./httpclient -dns-transport tcp ...")
-		fmt.Printf("%s\n", "  ./httpclient -dns-transport udp [-dns-udp-server <addr>:<port>] ...")
+		fmt.Printf("%s\n", "  ./httpclient -dns-server system:// ...")
+		fmt.Printf("%s\n", "  ./httpclient -dns-server godns:// ...")
+		fmt.Printf("%s\n", "  ./httpclient -dns-server https://<url> ...")
+		fmt.Printf("%s\n", "  ./httpclient -dns-server dot://<domain>|<addr>:<port> ...")
+		fmt.Printf("%s\n", "  ./httpclient -dns-server tcp://<addr>:<port> ...")
+		fmt.Printf("%s\n", "  ./httpclient -dns-server udp://<addr>:<port> ...")
```

Comment:
Same comment as above: it would be better to provide working-out-of-the-box examples